daemon/logdrivers_windows.go

```diff
@@ -0,0 +1,7 @@
+package daemon
```

Comment:
I guess it's not going to hurt anything since it's just importing the same thing again... nevermind.